### ISSUE TITLE
Areas of hypopigmentation and hyperpigmentation that follow Blaschko lines

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10921]
**Preferred term label:**

**Synonyms**
 hypo and hyperpigmentation following Blaschko lines

**Definition (free text, please give PubMed ID)**
PMID: 36316743 She presented pectus excavatum and 0.5  cm umbilical hernia. Hands with bilateral brachydactyly of the 5th finger and hallux valgus in both feet were also observed. In addition, pigmentary mosaicism with disseminated **hypo and hyperpigmentation following fne Blaschko lines** was observed.
(...) At pigmentation level, all three patients reported here showed fine Blaschko lines, been hyperpigmentation with disseminated dermatosis the most frequently observed pigmentation pattern. In previous studies of patients with pigmentary mosaicism, the hypopigmentation was the most frequent type of pigmentation (ranging 50–100%). The herein described patients share the hyperpigmentation pattern with a cohort of cases previously described by our group, who presented this characteristic in 77% of cases. In comparison, 7/21 previously reported patients with mosaic trisomy 12 showed pigmentary manifestations, described only as patchy or linear streaks. Unfortunately, only in 3 of the total previously reported cases, a detailed description of the pigmentation pattern with disseminated dermatosis following Blaschko lines was described.

**Parent term (use hpo.jax.org/app)**
Mixed hypo- and hyperpigmentation of the skin HP:0009123

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mosaic trisomy 12 ORPHA:1698

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001387
- label: Areas of hypopigmentation and hyperpigmentation that follow Blaschko lines
- parents: [Mixed hypo- and hyperpigmentation of the skin (HP:0009123)](https://hpo.jax.org/app/browse/term/HP:0009123)
- pmid: PMID:9125765
- def: Linear streaks of hypopigmentation or hyperpigmentation that are distributed according to Blaschko lines, which are V-shaped, S-shaped, or linear patterns on the skin that reflect the embryonic migration pathways of epidermal cells (keratinocytes, melanocytes, etc.) during early development.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10921
- synonyms:
